openmdao/visualization/connection_viewer/viewconns.py

```diff
@@ -46,7 +46,7 @@ def view_connections(root, outfile='connections.html', show_browser=True,
         The root for the desired tree.
 
     outfile : str, optional
-        The name of the output html file.  Defaults to 'connections.html'.
+        The name of the output file.  Defaults to 'connections.html'.
```

Comment:
should note here or in the summary that the output file format is determined by the file extension